src/Microsoft.SqlTools.ServiceLayer/ObjectManagement/ObjectTypes/ServerRole/ServerRoleViewContext.cs

```diff
@@ -0,0 +1,18 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+namespace Microsoft.SqlTools.ServiceLayer.ObjectManagement
+{
+    public class ServerRoleViewContext : SqlObjectViewContext
+    {
+        public ServerRoleViewContext(Contracts.InitializeViewRequestParams parameters) : base(parameters)
+        {
+        }
+
+        public override void Dispose()
+        {
```

Comment:
nothing to be disposed?

Reply:
For `ServerRole`, I didn't create a new connection (same behavior as Login) so I didn't dispose it here.